code-challenges/quick-sort/quickSort.test.js

```diff
@@ -0,0 +1,38 @@
+function quickSort (arr, left = 0, right = arr.length - 1) {
+  if (left < right) {
+    let position = partition(arr, left, right)
+    quickSort(arr, left, position - 1)
+    quickSort(arr, position + 1, right)
+  }
+  return arr
+}
+function partition (arr, left, right) {
+  let pivot = arr[right]
+  let low = left;
+  for (let i = left; i <= right; i++) {

```

Comment:
Hi Andrew,

Yes, the pseudo code has an error in it. It should say i from left to right - 1.
Try to use ```i < right ``` instead of ``` i <= right ```. See if your problem solved.

Ada